source/_docs/blueprint/tutorial.markdown

```diff
@@ -0,0 +1,229 @@
+---
+title: "Blueprint Tutorial"
+description: "Tutorial on creating a blueprint."
+---
+
+In this tutorial we're going to create a blueprint that contorls a light based on a motion sensor. We will do this by taking an existing automation and converting it to a blueprint.
+
+For this tutorial we use a simple automation. The process for converting a complex automation is not any different.
+
+## Our automation
+
+To create a blueprint, we first need to have a working automation with hardcoded values.
+
+The automation for this tutorial that controls a light based off a motion sensor looks like this:
+
+{% raw %}
+```yaml
+trigger:
+  platform: state
+  entity_id: binary_sensor.motion_kitchen
+
+action:
+  service: >
+    {% if trigger.to_state.state == "on" %}
+      light.turn_on
+    {% else %}
+      light.turn_off
+    {% endif %}
+  target:
+    entity_id: light.kitchen
+```
+{% endraw %}
+
+## Create the blueprint file
+
+All automation blueprints live in the `<config>/blueprints/automation/` folder and are defined as YAML files (with the `.yaml` extension). You can create as many subdirectories in this folder as you want.
+
+To get started with our blueprint, we're giong to copy the above automation YAML and save it as the file `<config>/blueprints/automation/motion_light_tutorial.yaml`.
+
+## Add basic blueprint metadata
+
+Home Assistant needs to know what your blueprint is about. This is done by adding a `blueprint:` section. It should contain the `domain` of the integration it is for (`automation`) and `name`, the name of your blueprint.
+
+Add this to the top of the file:
+
+```yaml
+blueprint:
+  name: Motion Light Tutorial
+  domain: automation
```

Comment:
```suggestion
  description: Turn a light on based on detected motion
  domain: automation
```

Reply:
I am not sure if we should include it in the example, because it takes up another line in already a long blueprint example.

Reply:
For me it made sense to show that option, because I do think users should provide a description to make using blueprints easier.
 
But I also had the same thought of keeping the examples lean, which is why I only proposed to add the description twice (once in the metadata section and once at the very end), but not in the YAML examples in between. We could add a short note for that fact as well in the paragraph (à la "the subsequent YAML examples will omit the description for brevity").

Reply:
I kinda agree with adding it. We should set the example here. When sharing blueprints, descriptions would make a difference for the consumer of such a blueprint.

Reply:
Yes, that makes sense. Will add.